cluster-autoscaler/cloudprovider/aws/ec2_instance_types.go

```diff
@@ -3735,27 +4148,6 @@ var InstanceTypes = map[string]*InstanceType{
 		GPU:          0,
 		Architecture: "amd64",
 	},
-	"p2.16xlarge": {
-		InstanceType: "p2.16xlarge",
-		VCPU:         64,
-		MemoryMb:     749568,
-		GPU:          16,
-		Architecture: "amd64",
-	},
-	"p2.8xlarge": {
-		InstanceType: "p2.8xlarge",
-		VCPU:         32,
-		MemoryMb:     499712,
-		GPU:          8,
-		Architecture: "amd64",
-	},
-	"p2.xlarge": {
-		InstanceType: "p2.xlarge",
-		VCPU:         4,
-		MemoryMb:     62464,
-		GPU:          1,
-		Architecture: "amd64",
-	},
```

Comment:
confirmed that all these instances are deprecated and not on the AWS docs anymore 👍🏾